inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -249,14 +239,23 @@ def positions(self, key, value):
     inst = {'name': value.get('a'),
             'record': inspire_dojson_utils.get_record_ref(recid,
                                                           'institutions')}
+
+    raw_rank = value.get('r')
+    if isinstance(raw_rank, tuple):
+        # FIXME: raise Exception("Multiple ranks at the same time?")
```

Comment:
I don't think that we should ever raise an `Exception` when running `dojson`. We should be logging the issue instead to `stderr`.